src/core.js

```diff
@@ -562,7 +562,7 @@ $.extend( $.validator, {
 			var lastActive = this.lastActive;
 			return lastActive && $.grep( this.errorList, function( n ) {
 				return n.element.name === lastActive.name;
-			} ).length === 1 && lastActive;
```

Comment:
You're welcome :)

> I agree we need to refactor things, because it is misleading

We have to look for a better refractoring (better than the one I provided above).

> But hey, we had unit tests which prevented that I am doing stupid things... ;-)

That's just the effect of JavaScript's trickiness :D